Jenkinsfile

```
@@ -927,17 +997,21 @@ pipeline {
                                                                statementCoverage: 0],
                                             ignore_failure: true
                     }
+                    post {
+                        cleanup {
+                            job_status_update()
+                        }
+                    }
                 } // stage('Bullseye Report')
             } // parallel
         } // stage ('Test Report')
     } // stages
     post {
         always {
-            valgrindReportPublish valgrind_stashes: ['centos7-gcc-nlt-memcheck',
-                                                     'centos7-gcc-unit-memcheck']
-        }
-        unsuccessful {
-            notifyBrokenBranch branches: target_branch
+          job_status_update('final_status')
+          job_status_write()
+          valgrindReportPublish valgrind_stashes: ['centos7-gcc-nlt-memcheck',
+                                                   'centos7-gcc-unit-memcheck']
         }
-    } // post
+     } // post
```

This looks like there is an extra space added unintentionally?
```suggestion
    } // post
```